docs/editor/settings-sync.md

```diff
@@ -144,37 +144,58 @@ Settings Sync uses a dedicated service to store settings and coordinate updates.
 
-Settings Sync persists authentication information to the system keychain. Writing to the keychain can fail in some cases if the keychain is misconfigured.
+> NOTE: keychain, keyring, wallet, credential store are synonomous in this document
+
+Settings Sync persists authentication information on desktop using the OS keychain for encryption. Using the keychain can fail in some cases if the keychain is misconfigured or the environment isn't recognized.
+
+To help diagnose the problem, the best thing to do is to first restart VS Code with the following flags:
+
+```
+code --verbose --vmodule="*/components/os_crypt/*=1"
+```
+
+### Windows & macOS
+
+At this time, there are no known configuration issues on Windows or macOS but if you suspect something is wrong [please open an issue on VS Code](https://github.com/microsoft/vscode/issues/new/choose) with the verbose logs from above. This is important for us to support additional desktop configurations.
 
 ### Linux
 
-#### Encryption not available
+Towards the top of the logs from the previous command, you will see something to the effect of:
+
+```
+[9699:0626/093542.027629:VERBOSE1:key_storage_util_linux.cc(54)] Password storage detected desktop environment: GNOME
+[9699:0626/093542.027660:VERBOSE1:key_storage_linux.cc(122)] Selected backend for OSCrypt: GNOME_ANY
+```
+
+We rely on Chromium's oscrypt module to descover and store encryption key information in the keyring. Chromium supports [a number of different desktop environments](https://source.chromium.org/chromium/chromium/src/+/main:base/nix/xdg_util.cc;l=146-169). Outlined below are some popular desktop environments and some troubleshooting steps that may help if the keyring is misconfigured.
+
+#### GNOME or UNITY (or similar)
+
+If the error you're seeing is "Cannot create an item in a locked collection", chances are your keyring's `Login` keyring is locked. You should launch your OS's keyring (seahorse is the commonly used GUI for seeing keyrings) and ensure the default keyring (usually referred to as `Login` keyring) is unlocked. This keyring needs to be unlocked when you log into your system.
+
+#### KDE
+
+It's possible that your wallet (aka keyring) is closed. If you open KWalletManager, you can see if the default `kdewallet` is closed and if it is, make sure you open it.
 
-If you are experiencing issues with encryption not being available, please follow the steps below:
+#### Other Linux desktop environments
 
-1. Check if the environment variable `XDG_CURRENT_DESKTOP` is defined. You can do this by opening a terminal and running the following command:
+First off, if your desktop environment wasn't detected, [please open an issue on VS Code](https://github.com/microsoft/vscode/issues/new/choose) with the verbose logs from above. This is important for us to support additional desktop configurations.
 
-   ```bash
-   echo $XDG_CURRENT_DESKTOP
-   ```
+##### (recommended) Configure the keyring to use with VS Code
```

Comment:
Is this really only for the KDE section, or does it apply to other desktop environments as well?
I'm also unsure whether the site renders h5's properly.

Reply:
It's under the `Other Linux desktop environments` section.

Reply:
If h5's don't work I guess I could just bold them

Reply:
It is supported.
I'll create an item to bold them on my side.
Ref https://code.visualstudio.com/docs/java/java-refactoring#_convert-to-lambda-expression